src/cli/cert.rs

```diff
@@ -66,6 +82,27 @@ pub fn run(cert_args: CertArgs) -> exitcode::ExitCode {
                 println!("{}", serde_json::to_string(&success_msg).unwrap());
             };
         }
+        CertCommands::Upload(upload_args) => {
+            let api_key = get_api_key!();
+            let cert_ref =
+                match cert::upload_new_cert_ref(&upload_args.cert_path, &api_key, upload_args.name)
+                    .await
+                {
+                    Ok(pcr8) => pcr8,
+                    Err(e) => {
+                        log::error!(
+                            "An error occurred while generating PCR8 for your cert - {}",
+                            e
+                        );
+                        return e.exitcode();
+                    }
+                };
+            let success_msg = serde_json::json!({
+                "status": "success",
+                "output": cert_ref,
+            });
+            println!("{}", serde_json::to_string(&success_msg).unwrap());
```

Comment:
Should there be a human readable output if stdout isn't being piped here?

Reply:
There should, updated 👍